### PR TITLE
Improve invalidation with concurrency limit

### PR DESCRIPTION
**Pull request by PabloSzx**
Trying to invalidate with redis filled with data breaks the invalidation logic because of calling a Promise.all with too much data

### CODE REVIEW

File: src/redis.ts [hunk 1]
```diff
@@ -87,6 +88,11 @@ export type RedisCacheParameter = {
   GETRedisTimeout?: number;
 
   onError?: (err: unknown) => void;
+
+  /**
+   * @default 10
```
Comment (PabloSzx) [inline src/redis.ts:93]: is default 10 too low? 🤔

File: src/utils.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export function chunk<T>(array: T[], size: number) {
```
Comment (saihaj) [inline src/utils.ts:1]: should add  a basic tests for this chunk function

File: src/redis.ts (hunk 1 above)
Comment (vigneshka) [reply src/redis.ts:93]: Yeah, could do 100

File: src/utils.ts (hunk 2 above)
Comment (PabloSzx) [reply src/utils.ts:1]: https://github.com/soundxyz/response-cache/pull/41/commits/757d5b2024f157677db521aba0a73be66899e528

**Review by saihaj — APPROVED** [review]

File: src/redis.ts (hunk 1 above)
Comment (PabloSzx) [reply src/redis.ts:93]: I will put default at 20, libraries should have safe defaults

**Pull request merged**